package-lock.json

```diff
@@ -1074,6 +1074,14 @@
         "semver": "^5.5.0",
         "shebang-command": "^1.2.0",
         "which": "^1.2.9"
+      },
```

Comment:
Looks like these changes are the result of different environments. I usually recommend to unstage these as otherwise we'll bounce hence and forth with every single commit.

Reply:
Okay good to know.